s3-test/src/main/java/com/hazelcast/jet/tests/s3/S3WordCountTest.java

```diff
@@ -133,6 +134,20 @@ private boolean isSocketTimeoutException(Throwable e) {
         return false;
     }
 
+    private boolean isSocketException(Throwable e) {
```

Comment:
i think it would be easier to merge this with the isSocketTimeoutException method?

Reply:
sure